app/controllers/users.js

```diff
@@ -8,7 +8,15 @@ exports.signUp = (req, res, next) => {
     .catch(next);
 };
 
-exports.signIn = async (req, res) => {
-  const user = await findUserByEmail(req.body.email);
-  res.send({ accessToken: generateToken(user) });
+exports.signIn = (req, res, next) => {
+  findUserByEmail(req.body.email)
+    .then(user => res.send({ accessToken: generateToken(user) }))
+    .catch(next);
+};
+
+exports.index = (req, res, next) => {
```

Comment:
why is it called index?

Reply:
I don't know, is just a convention I got from other projects. Every list of models is commonly called an "index of " (users, in this case). Would you say it would be better if I change it to `getAllUsers` ?